<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QDialog" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="picButton1">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>115</width>
     <height>115</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="picButton2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>115</width>
     <height>115</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="picButton4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>115</width>
     <height>115</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="picButton5">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>400</y>
     <width>115</width>
     <height>115</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="picButton3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>280</y>
     <width>115</width>
     <height>115</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="picButton6">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>400</y>
     <width>115</width>
     <height>115</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sureButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>530</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>чбошод</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
